client/lib/wpcom-undocumented/lib/undocumented.js

```diff
@@ -1455,6 +1455,26 @@ Undocumented.prototype.usersEmailNew = function( query, fn ) {
 	return this.wpcom.req.post( args, fn );
 };
 
+/**
+ * Sign up for a new user account and login immediately
+ * Onboard a new user
+ *
+ * @param {object} query - an object with the following values: email
+ * @param {Function} fn - Function to invoke when request is complete
+ */
+Undocumented.prototype.usersEmailOnboard = function( query, fn ) {
```

Comment:
This function name does not make it obvious to me that we're creating a user account as a byproduct.  How about something more like `createUserAccountFromEmailAddress` (or something)?